app/services/fcm_pushalarm.py

```
@@ -71,7 +71,8 @@ def init(self):
             logger.error("[FCM] 초기화 실패: %s", e)
 
     async def send_notification(self, device_token: str, body: str,
-                                from_user_id: str = None, level: str = None) -> bool:
+                                from_user_id: str = None, level: str = None,
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)